playbooks/README.md

```diff
@@ -0,0 +1,21 @@
+# openshift-ansible playbooks
+
+In summary:
+
+- [`byo`](byo) (_Bring Your Own_ hosts) has the most actively maintained
+  playbooks for installing, upgrading and performing others tasks on OpenShift
+  clusters.
+
+And:
+
+- [`adhoc`](adhoc) is a generic home for playbooks and tasks, unsupported and
+  not maintained.
```

Comment:
s/unsupported and not maintained/community supported and not officially maintained/